zerver/tests/test_signup.py

```diff
@@ -3596,34 +3592,30 @@ def test_find_team_zero_emails(self) -> None:
     def test_find_team_one_email(self) -> None:
         data = {'emails': self.example_email("hamlet")}
         result = self.client_post('/accounts/find/', data)
-        self.assertEqual(result.status_code, 302)
-        self.assertEqual(result.url, '/accounts/find/?emails=hamlet%40zulip.com')
+        self.assertEqual(result.status_code, 200)
         from django.core.mail import outbox
         self.assertEqual(len(outbox), 1)
 
     def test_find_team_deactivated_user(self) -> None:
         do_deactivate_user(self.example_user("hamlet"))
         data = {'emails': self.example_email("hamlet")}
         result = self.client_post('/accounts/find/', data)
-        self.assertEqual(result.status_code, 302)
-        self.assertEqual(result.url, '/accounts/find/?emails=hamlet%40zulip.com')
+        self.assertEqual(result.status_code, 200)
         from django.core.mail import outbox
         self.assertEqual(len(outbox), 0)
 
     def test_find_team_deactivated_realm(self) -> None:
         do_deactivate_realm(get_realm("zulip"))
         data = {'emails': self.example_email("hamlet")}
         result = self.client_post('/accounts/find/', data)
-        self.assertEqual(result.status_code, 302)
-        self.assertEqual(result.url, '/accounts/find/?emails=hamlet%40zulip.com')
+        self.assertEqual(result.status_code, 200)
         from django.core.mail import outbox
         self.assertEqual(len(outbox), 0)
 
     def test_find_team_bot_email(self) -> None:
         data = {'emails': self.example_email("webhook_bot")}
         result = self.client_post('/accounts/find/', data)
-        self.assertEqual(result.status_code, 302)
-        self.assertEqual(result.url, '/accounts/find/?emails=webhook-bot%40zulip.com')
+        self.assertEqual(result.status_code, 200)
```

Comment:
We should change these to validate the HTML of `result` (there's some test helper methods for this purpose) to verify that the email address appears in the rendered content.

Reply:
Sounds reasonable, will do so :smile: